eva/udfs/decorators/yolo_object_detection_decorators.py

```diff
@@ -46,7 +46,7 @@ def name(self) -> str:
     @setup(cachable=True, udf_type="object_detection", batchable=True)
     def setup(self, threshold=0.85):
         self.threshold = threshold
-        self.model = torch.hub.load("ultralytics/yolov5", "yolov5s", verbose=False)
+        self.model = torch.hub.load("ultralytics/Yolo", "Yolos", verbose=False)
```

Comment:
Also, change this to new code. Btw, can you get rid of older code and let us only use decorators?

Reply:
Yes done